src/UserAdmin/src/Smart.FA.Catalog.Web/Pages/SuperUser/Trainers/SuperUserTrainerList.cshtml.cs

```diff
@@ -0,0 +1,118 @@
+﻿using MediatR;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+using Microsoft.AspNetCore.Mvc.ViewFeatures;
+using Microsoft.Extensions.Options;
+using Smart.Design.Razor.TagHelpers.Alert;
+using Smart.FA.Catalog.Application.UseCases.Commands;
+using Smart.FA.Catalog.Application.UseCases.Queries;
+using Smart.FA.Catalog.Core.Domain;
+using Smart.FA.Catalog.Core.Services;
+using Smart.FA.Catalog.Shared.Collections;
+using Smart.FA.Catalog.Web.Options;
+
+namespace Smart.FA.Catalog.Web.Pages.SuperUser.Trainers;
+
+public class List : PageModel
+{
+    [BindProperty(SupportsGet = true)]
+    public string? TrainerNameOrEmailFilterQuery { get; set; }
+
+    [BindProperty(SupportsGet = true)]
+    public int CurrentPage { get; set; } = 1;
+
+    public IUserIdentity UserIdentity { get; }
+
+    public IMediator Mediator { get; }
+
+    public SuperUserOptions Settings { get; set; }
+
+    public PagedList<Trainer> TrainerList { get; set; }
+
+    public List(IUserIdentity userIdentity, IMediator mediator, IOptions<SuperUserOptions> superUserOptions)
+    {
+        UserIdentity = userIdentity;
+        Mediator = mediator;
+        Settings = superUserOptions.Value;
+    }
+
+    public async Task OnGetAsync()
+    {
+        // Display side panel as selected
+        ViewData[nameof(SuperUserSideMenuItem)] = SuperUserSideMenuItem.SuperUserTrainerList;
+
+        // Get All trainers (except yourself) paged
```

Comment:
Yourself would refer to the reader of the code I guess :p

```suggestion
        // Get All trainers (except current connected user) paged
```

or

```suggestion
        // Get All trainers (but current connected user) paged
```

or keep it that way.